### PR TITLE
AU Core Future Considerations

### PR DESCRIPTION
**Pull request by dt-r**
Added a set of future considerations for AU Core. This is:

1. non-exhaustive
2. includes 'agreed' future considerations as voting on in meetings incl AUCDI R1 not yet supported
3. includes expected R2 items
4. includes likely items coming from eRequesting
5. includes big ticket items (e.g. write & update) important to note are future considerations

### CODE REVIEW

File: input/pagecontent/future.md [hunk 1]
```diff
@@ -34,8 +34,18 @@ The approach to develop and mature AU Core is outlined below making reference to
...
+* **AU Core MedicationStatement**: Development and testing of AU Core MedicationStatement profile to support AUCDI Medication Use Statement is planned for the AU Core R2 scope.
+* **Dosage**: The Dosage data type is complex, and the requirements in AU Base Dosage are not sufficient to provide clarity on the minimum requirements for supporting dosage and timing in Australia. Future versions of AU Core will develop and test an AU Core Dosage profile to support common implementation of medicine information. 
+* **Composition**: Future versions of AU Core will develop and test AU Core Composition profile to support approaches to sharing clinical documents across multiple use cases.
+* **DocumentReference**: Future versions of AU Core will develop and test approaches and AU Core DocumentReference profile to support sharing of clinical notes across multiple use cases to support common implementation.
```
Comment (JayMurdoch) [inline input/pagecontent/future.md:41]: Suggest: "Future versions of AU Core will develop and test approaches **to** AU Core DocumentReference profile..."

File: input/pagecontent/future.md [hunk 2]
```diff
@@ -34,8 +34,18 @@ The approach to develop and mature AU Core is outlined below making reference to
...
+* **RelatedPerson**: Development and testing of AU Core RelatedPerson profile to support the exchange of core digital health and administrative information that involves persons with a personal relationship or non-healthcare-specific professional relationship to the patient is planned for the AU Core R2 scope.
+* **HealthcareService**: Development and testing of AU Core HealthcareService profile to define minimum requirements for support health care service information across multiple use cases to support common implementation. Work will also be undertaken to consider guidance on resource selection across HealthcareService, PractitionerRole, Organization, and Location.
```
Comment (JayMurdoch) [inline input/pagecontent/future.md:43]: Suggest: "...minimum requirements for **the** support **of healthcare** service information..."

File: input/pagecontent/future.md [hunk 3]
```diff
@@ -34,8 +34,18 @@ The approach to develop and mature AU Core is outlined below making reference to
...
+* **Device**: Use of the Device resource for medical devices, and devices that perform, observe, or author clinical data, are required to support multiple downstream use cases including eRequesting and patient summary. Future versions of AU Core will develop and test profiles to support common implementation of Device information.
+* **Coverage**: Use of the Coverage resource is required to support multiple downstream use cases including eRequesting. Future versions of AU Core will develop and test minimum requirements for coverage across multiple use cases to support common implementation.
+* **ServiceRequest**: Use of the ServiceRequest resource is required to support multiple downstream use cases including eRequesting. Future versions of AU Core will develop and test minimum requirements for requests across multiple use cases to support common implementation.
+* **Diagnostic Results**: Additional AU Core profiles to support common implementation of additional aspects of diagnostic results including imaging, specimen, and reports will be developed and test in future versions of AU Core.
```
Comment (JayMurdoch) [inline input/pagecontent/future.md:47]: Suggest: "...developed and **tested** in future versions of AU Core."

File: input/pagecontent/future.md [hunk 4]
```diff
@@ -34,8 +34,18 @@ The approach to develop and mature AU Core is outlined below making reference to
...
+* **Consideration whether some clinical elements should not allow Missing Data**: It is not clear at this point whether there is a case for prohibiting 'missing data' and 'suppressed data' on resource.code e.g. AllergyInterolance.code, Condition.code, or Medication.code. At this time it is not clear how prohibition could be structured in AU Core noting there is no equivalent obligation code for 'don't allow missing data' and as the data type for these elements is CodeableConcept, preadoption of R5 mustHaveValue is not applicable (only for primitive types).
```
Comment (JayMurdoch) [inline input/pagecontent/future.md:48]: Suggest: "Consideration **of** whether some..."

**Review by JayMurdoch — CHANGES_REQUESTED** [review]
Suggested changes outlined in inline comments.

File: input/pagecontent/future.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/future.md:41]: agree wording change is needed but we don't want to 'to a profile'. we want to develop and test a profile.

Comment (dt-r) [reply input/pagecontent/future.md:41]: fixed wording

File: input/pagecontent/future.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/future.md:43]: changed wording, bit different to what was suggested.

File: input/pagecontent/future.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/future.md:47]: fixed.

File: input/pagecontent/future.md (hunk 4 above)
Comment (dt-r) [reply input/pagecontent/future.md:48]: Grammatical correctness of the bolding isn't desired. Re-reading it I want to shorten it further.

**Review by dbojicic — APPROVED** [review]

**Pull request merged**